_rules/aria-state-or-property-allowed-5c01ea.md

```diff
@@ -5,11 +5,12 @@ rule_type: atomic
 description: |
   This rule checks that WAI-ARIA states or properties are allowed for the element they are specified on.
 accessibility_requirements:
-  wcag20:4.1.2: # Name, Role, Value (A)
+  aria11:supportedState:
+    title: ARIA 1.1, 5.2.3 Supported States and Properties
```

Comment:
When looking at the rule, it tests whether the state/property is EITHER [inherited](https://www.w3.org/TR/wai-aria/#inheritedattributes), [supported](https://www.w3.org/TR/wai-aria/#supportedState), or [required](https://www.w3.org/TR/wai-aria/#requiredState), so I don't think it's enough to link to one of them as the requirement...

Reply:
I disagree. It seems to me that inherited properties, and required properties are a subset of supported properties. If you have a better suggestion that would be appreciated.

Reply:
@WilcoFiers , @kasperisager says you are right.

Reply:
... but we strongly suggest we link to "https://www.w3.org/TR/wai-aria-1.1/#states_and_properties" instead, where the whole thing is defined.

Reply:
Come to think of it, I'd actually suggest we associate the rule with https://www.w3.org/TR/wai-aria-1.1/#state_property_processing given that it defines a "**MUST**" author conformance requirement:

> However, authors **MUST** only use non-global states and properties on elements with a role supporting the state or property; either defined as an explicit WAI-ARIA role, or as defined by the host language implicit WAI-ARIA semantic matching an appropriate WAI-ARIA role.